packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -1,90 +1,185 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+/* stylelint-disable max-nesting-depth */
 // Themes
 @import './clinical-lowlight-theme/DatePicker.module';
 @import './orion-fusion-theme/DatePicker.module';
 
 :local {
   .date-picker {
     display: inline-block;
+  }
+
+  .date-input-container {
+    display: inline-flex;
+  }
+
+  .date-input {
+    background-color: var(--terra-date-picker-input-background-color, #fff);
+    border-bottom-right-radius: 0;
+    border-color: var(--terra-date-picker-input-border-color, #dedfe0);
+    border-radius: var(--terra-date-picker-input-border-radius, 3px);
+    border-style: var(--terra-date-picker-input-border-style, solid);
+    border-top-right-radius: 0;
+    border-width: var(--terra-date-picker-input-border-width, 1px);
+    box-sizing: content-box;
+    color: var(--terra-date-picker-input-color, #1c1f21);
+    display: inline-block;
+    padding-bottom: var(--terra-date-picker-input-padding-bottom, 0.2857142858rem);
+    padding-left: var(--terra-date-picker-input-padding-left, 0.2857142858rem);
+    padding-right: var(--terra-date-picker-input-padding-right, 0.2857142858rem);
+    padding-top: var(--terra-date-picker-input-padding-top, 0.2857142858rem);
+    vertical-align: middle;
+    white-space: nowrap;
+    width: 8em;
+
+    &.is-focused {
+      border-color: var(--terra-date-picker-input-focus-border-color, #26a2e5);
+      box-shadow: var(--terra-date-picker-input-focus-box-shadow, none);
+      outline: none;
+      z-index: 1;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-date-picker-input-disabled-background-color, rgba(111, 116, 119, 0.2));
+      border: var(--terra-date-picker-input-disabled-border, 1px solid #868a8c);
+      color: var(--terra-date-picker-input-disabled-color, #1c1f21);
+      cursor: default;
+      opacity: var(--terra-date-picker-input-disabled-opacity, 0.25);
```

Comment:
Should we be adding opacity? Wasn't this causing accessibility issues?

Reply:
I am not aware of the accessibility issue caused by its use, also i referenced `time-input` for the property value for consistency.
cc: @neilpfeiffer